lib/include/customer.h

```diff
@@ -3,13 +3,15 @@
 #include <cstdint>
 #include <utility>
 #include <vector>
+#include <transportation_quantity.h>
 
 namespace vrp {
 /// Customer representation
 class Customer {
 public:
     int id = 0; ///< customer id. 0 is reserved for depot
     int demand = 0;    ///< customer demand
+    TrQuant trquant_c; ///customer demand quantity (of 1 item)
```

Comment:
i'd just name the member variable `quantity`

Reply:
by the way, this is not a quantity **for 1 item**. This is a **total** quantity needed by customer (as far as I understand)

Reply:
There should be no "int demand".
Instead it should be "TransportationQuantity demand".
Yes, it is total quantity.

Reply:
agree with Mike